### PR TITLE
[HSEARCH] 6.2.0.CR1

### PR DESCRIPTION
**Pull request by yrodiere**

### CODE REVIEW

**Comment by yrodiere** [comment]
Should be available soon at https://staging.in.relation.to/2023/06/26/hibernate-search-6-2-0-CR1/

File: posts/Yoann/2023-06-26-hibernate-search-6-2-0-CR1.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+for more information.
+
+[[other-changes]]
+=== Other improvements and bug fixes
+
+* link:{hsearch-jira-url-prefix}/HSEARCH-4616[HSEARCH-4616]:
+For clarity, "automatic indexing" was renamed to something more explicit throughout Hibernate Search and its documentation.
+As a result:
+** `hibernate.search.automatic_indexing.enabled` is now deprecated in favor of `hibernate.search.indexing.listeners.enabled`
+** `hibernate.search.automatic_indexing.synchronization.strategy` is now deprecated in favor of `hibernate.search.indexing.plan.synchronization.strategy`
```
Comment (marko-bekhta) [inline posts/Yoann/2023-06-26-hibernate-search-6-2-0-CR1.adoc:78]: We just add it here for completeness right ( I think we've released this one in a previous version)?

**Review by marko-bekhta — APPROVED** [review]
Looks good 😃 🏆

File: posts/Yoann/2023-06-26-hibernate-search-6-2-0-CR1.adoc (hunk 1 above)
Comment (yrodiere) [reply posts/Yoann/2023-06-26-hibernate-search-6-2-0-CR1.adoc:78]: Yep, this felt relevant.

**Pull request merged**